setup.py

```diff
@@ -129,7 +129,9 @@ def run(self):
         'testing': [
             'fields',
             'hunter',
+            # TODO: unpin?!
             'process-tests==2.0.0',
+            # TODO: unpin?!
```

Comment:
So the idea was that things we test against (iow: we expect regressions or integration issues on those) need to be pinned so we have a predictable test suite. That way we're able to say: this pytest-cov release is compatible with pytest-xdist==X.Y.Z

If we don't have pins then things can get pretty random.

Also: https://caremad.io/posts/2013/07/setup-vs-requirement/ ('s/requirement.txt/tox.ini/`)

Pins shouldn't exist in setup.py as it can break user install scenarios.

Reply:
Those are only for extras_require=testing.

Reply:
Addressed in 735c720.